core/templates/pages/exploration-editor-page/feedback-tab/thread-table/thread-table.component.html

```diff
@@ -15,7 +15,7 @@
       <th>Last updated</th>
     </tr>
   </thead>
-  <tr *ngFor="let thread of this.threads" (click)="onClickRow(thread.threadId)" class="oppia-feedback-tab-row protractor-test-oppia-feedback-tab-row">
+  <tr *ngFor="let thread of this.threads" (click)="clickRowEventEmitter.emit(thread.threadId)" class="oppia-feedback-tab-row protractor-test-oppia-feedback-tab-row">
```

Comment:
Maybe create a function called onRowClick keep `clickRowEventEmitter.emit(thread.threadId)` inside the function.

Reply:
Done.